### PR TITLE
Detecting geometrical shapes using opencv

### PR DESCRIPTION
**Pull request by simarpreetsingh-019**
Detecting geometrical shapes using opencv in python added.

This Pr is in reference to issue #93  being assigned to me with label as Medium and CH20. would create readme also in some time.
Please review.
https://github.com/ankitdobhal/Awesome-Python-Scripts/issues/93#issuecomment-682343103

### CODE REVIEW

File: Image-Processing/Detecting Contours/detecting_geometrical_shapes.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        cv2.putText(img, "star", (x, y), cv2.FONT_HERSHEY_COMPLEX, 0.5, (0, 0, 0))
+     
+    else:
+        cv2.putText(img, "circle", (x, y), cv2.FONT_HERSHEY_COMPLEX, 0.5, (0, 0, 0))
+
+cv2.imshow('shapes', img)
+cv2.waitKey(0)
+cv2.destroyAllWindows()
+
+
```
Comment (disha2sinha) [inline Image-Processing/Detecting Contours/detecting_geometrical_shapes.py]: Remove new line

**Review by disha2sinha — CHANGES_REQUESTED** [review]
Prepare a README.md file with brief description of your work,libraries used and outputs

File: Image-Processing/Detecting Contours/detecting_geometrical_shapes.py (hunk 1 above)
Comment (simarpreetsingh-019) [reply Image-Processing/Detecting Contours/detecting_geometrical_shapes.py]: okay, will be updating by morning on all these.

**Review by disha2sinha — CHANGES_REQUESTED** [review]
Remove issue #246 and link this PR to issue #93

**Comment by simarpreetsingh-019** [comment]
@disha2sinha only the ones with admin access can remove or delete that issue 246, so I am not able to do so, maybe you or someone with admin rights can do that.

**Review by disha2sinha — APPROVED** [review]
Yeah by remove I meant close.....my fault

**Comment by simarpreetsingh-019** [comment]
ohh , no problem, I have already closed them.
is it ready for merge?

**Pull request merged**